src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/JudgeApprovedOrderAdditionalDocumentsCollection.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class JudgeApprovedOrderAdditionalDocumentsCollection {
```

Comment:
not sure if you can reuse `uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.draftorders.CaseDocumentCollection`

Reply:
Perhaps, move `CaseDocumentCollection` to `uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd` package